Team.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <iostream>
+#include <string>
+#include <iomanip>
+
+class Team
+{
+public:
+  std::string name;
+  double fifaPoints, fifaBefore, startingFifaPoints;
+  int fifaRank, place, games, points, goalsScored, goalsConceded, goalsDifference;
+  Team(std::string nameNew, int fifaRankNew, double fifaPointsNew);
+  ~Team() = default;
+  const std::tuple<int, int, int, double> evaluatingStats(int scored, int conceded, double importanceOfMatch, double anotherTeamPoints);
```

Comment:
Better return a custom struct, because fields in a struct are named, while it's impossible to understand what unnamed members of a tuple mean.

Reply:
Also 'ing' in the name is confusing, why not evaluateStats?